### DISCUSSION TITLE
Question : how to make popup go on top of powergrid layout

### DISCUSSION THREAD

**Discussion by BenOussama180** [Power-Components/livewire-powergrid#793] [General]
so i have these popups/dropdowns inside my powergrid table : 
![image](https://user-images.githubusercontent.com/81111919/210958554-91dd2816-f0f7-4f13-8284-582be9441fb6.png)
![image](https://user-images.githubusercontent.com/81111919/210958585-a3b72751-5794-47e7-8d91-c1d5bffe573a.png)
as you can see they are under powergrid layout, i want them to be at the TOP to be shown everytime everywhere.
**yes i tried z-50 not working, i tried playing with positions not working also**
@luanfreitasdev any idea? thanks in advanced.

**Comment by luanfreitasdev** [comment]
I think this is not possible because of the current table structure. I have tried a few things to achieve this but have not been successful. In that case, what you could do is have some mechanism that makes the content display upwards

I'm still looking for a solution for this

**Reply by BenOussama180** [reply]
hmm well besides switching to modals i dont know what could  be done
